praw/objects.py

```diff
@@ -1807,8 +1809,9 @@ def __init__(self, reddit_session, subreddit=None, page=None,
             page = json_dict['page']
         info_url = reddit_session.config['wiki_page'].format(
             subreddit=six.text_type(subreddit), page=page)
+        self._params = kwargs
```

Comment:
This would overwrite the unique count, so you'd need to move the unique count to after here if this change sticks.